Gemfile

```diff
@@ -51,6 +51,8 @@ unless ENV["PACKAGING"] && ENV["PACKAGING"] == "yes"
     gem "rubocop", "~> 0.46", require: false
     gem "brakeman", require: false
     gem "database_cleaner"
+    gem "pry"
+    gem "pry-nav"
```

Comment:
I asked as couple of days ago and I think @mssola agreed. I use the regularly and I have to remember to skip them when indexing changes in git. They live only in :development and :test so it shouldn't be much of a problem. I just have to push the docker image for dev env.